app/assets/javascripts/browsered/web-payments/helpers.js

```diff
@@ -10,6 +10,7 @@ const showErrorSummary = (title, body) => {
   errorSummary.classList.remove('hidden')
   errorSummary.querySelectorAll('h2')[0].innerText = title
   if (body) {
+    errorSummary.querySelectorAll('ul')[0].innerHTML = ''
```

Comment:
Maybe we should clear the contents of the `ul` even if there is no body. I imagine if we called this method with just a title, we would want just that title to be displayed and not the body of the previous error.

Reply:
Good point. I've now made that change.